sys/net/gnrc/routing/rpl/gnrc_rpl_control_messages.c

```diff
@@ -737,121 +742,186 @@ void gnrc_rpl_recv_DIS(gnrc_rpl_dis_t *dis, kernel_pid_t iface, ipv6_addr_t *src
     }
 }
 
-void gnrc_rpl_recv_DIO(gnrc_rpl_dio_t *dio, kernel_pid_t iface, ipv6_addr_t *src, ipv6_addr_t *dst,
-                       uint16_t len)
+/**
+ * @brief   Handles the options from a received DIO packet.
+ *
+ * @param[in] inst      The @p RPL instance that the DIO belongs to.
+ * @param[in] dio       The @p DIO packet.
+ * @param[in] src       The address of the sender.
+ * @param[in] len       The length of the whole DIO packet.
+ * @param[in] is_new    Whether the DIO belongs to an existing or newly created DODAG.
+ *
+ * @returns             True, on success.
+ * @returns             False, if parsing of the options failed.
+ * @returns             False, if the DODAG is new and the GNRC_RPL_OPT_DODAG_CONF option is required but missing.
```

Comment:
```suggestion
 * @retval              True, on success.
 * @retval              False, if parsing of the options failed.
 * @retval              False, if the DODAG is new and the GNRC_RPL_OPT_DODAG_CONF
                        option is required but missing.
```

The comment for documenting the return values is `@retval`. You can only have one `@return` or `@returns` command per function.

Also the last line is longer than 100 characters.

Reply:
> The comment for documenting the return values is `@retval`. You can only have one `@return` or `@returns` command per function.

Ah okay. Did that change? I saw quite a lot of `@return{s}`s with multiple values throughout the documentation, e.g. in the RPL or netapi docs.